README.rst

```diff
@@ -74,11 +74,13 @@ Optionally:
 - zeroconf (for HiSLIP and VICP devices discovery)
 - pyvicp (to enable the Teledyne LeCroy proprietary VICP protocol)
 
+Please refer to `pyproject.toml <./pyproject.toml>`_ for the specific version
+requirements.
```

Comment:
I'm not sure if this approach I followed works okay for PyPI page, though. GitHub properly redirects to the repository file, but I cannot see how PyPI would do the same.

Let me know if you think that's an issue.

Reply:
You can use `twine check` on a sdist to check if the README is admissible for PyPI. I will do it next week if you do not beat me to it.

Reply:
Nice. I didn't know about `twine check`. Thanks for checking this out.